app/controllers/application_controller.rb

```diff
@@ -8,8 +8,15 @@ class ApplicationController < ActionController::Base
 
   before_action :log_active_connections
 
+  around_action :trace_span
+
   private
 
+  def trace_span(&block)
+    Google::Cloud::Trace
```

Comment:
Is there anything we need to install / setup to get activerecord tracing? I'm not sure how that works.

Reply:
I think we get that out of the box here: https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud-trace/lib/google/cloud/trace/rails.rb#L73-L79